### PR TITLE
fix changelog

### PR DESCRIPTION
**Pull request by krmayankk**
Release Note
```release-note
none
```

### CODE REVIEW

**Comment by krmayankk** [comment]
@tallclair @liggitt

**Comment by jpiper** [comment]
Also might be worth adding in the `--tls-cipher-suites` flag added in #48859 which was missed from the release notes for some reason

File: CHANGELOG-1.10.md [hunk 1]
```diff
@@ -574,6 +574,7 @@ And point to the file using the --config flag, as in
   * [stable] [containerd](https://github.com/containerd/containerd): containerd v1.1 natively supports CRI v1alpha2 now, so users can use Kubernetes v1.10 with containerd v1.1 directly, without having to use the intermediate cri-containerd daemon. [All Kubernetes 1.10 tests passed](https://k8s-testgrid.appspot.com/sig-node-containerd). [@Random-Liu]
   * [stable] [cri-o](https://github.com/kubernetes-incubator/cri-o): cri-o v1.10 updated CRI version to v1alpha2 and made several bug and stability fixes. [@mrunalp]
   * [stable] [frakti](https://github.com/kubernetes/frakti): frakti v1.10 implemented GCE Persistent Disk as a high performance volume, fixed several bugs, added ARM64 support, and passed all CRI validation conformance tests and node e2e conformance tests. [@resouer]
+  * [alpha] CRI now supports specifying the GID of the container at both LinuxSandboxSecurityContext and LinuxContainerSecurityContext in addition to specifying the UID. Thi    support is only implemented for dockershim and other CRI's are welcome to implement it. [@krmayankk]
```
[the remark this answers is not in the document]
Comment (krmayankk) [reply CHANGELOG-1.10.md]: fixed

**Comment by krmayankk** [comment]
@jpiper i dont know enough about this change. If you want to tell me exactly what to include i can include in the changelog.

**Comment by krmayankk** [comment]
@liggitt  can you take a look ?

**Comment by jpiper** [comment]
@krmayankk under `--tls-min-version on kubelet and kube-apiserver allow for configuring minimum TLS versions` please put

```
* kube-apiserver and kubelet now support customizing TLS ciphers via a `--tls-cipher-suites` flag ([#48859](https://github.com/kubernetes/kubernetes/pull/48859), [@victorgp](https://github.com/victorgp))
```

**Comment by krmayankk** [comment]
@jpiper its hard to get lgtm's ;-) , can you make another pr ?

**Comment by jpiper** [comment]
@krmayankk sure thing

**Pull request merged**